### PR TITLE
Better README.md

### PR DESCRIPTION
**Pull request by JollyJolli**
Hope this help :)

solves #33

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,59 +1,96 @@
...
+npm install
+```
 
-## To start the server side development locally in docker container
+### 4. Create an Environment File
+Create a `.env` file in the `client` directory and add the following lines:
+```env
+REACT_APP_PUBLIC_FOLDER=http://localhost:8000/images/
+GENERATE_SOURCEMAP=false
+PATH=http://localhost:8000
```
Comment (artizticamit) [inline README.md]: Could you please remove this line "PATH=http://localhost:8000". This was added by mistake. 
Thanks

Comment (artizticamit) [reply README.md]: @JollyJolli

Comment (JollyJolli) [reply README.md]: Yes of course!

Comment (JollyJolli) [reply README.md]: What do i put?

Comment (artizticamit) [reply README.md]: nothing just need to remove that part

Comment (JollyJolli) [reply README.md]: All the ENV or only the localhost?

Comment (artizticamit) [reply README.md]: only this line "PATH=http://localhost:3000", all other ENV var should remain

Comment (JollyJolli) [reply README.md]: Just commit the changed of removing the PATH

**Review by artizticamit — APPROVED** [review]

**Pull request merged**